### ISSUE TITLE
how to open tensorcore in TFS1.13

### ISSUE THREAD

**Issue by wangyunxiaa** [tensorflow/tensorflow#32458]
I have a language model, which has some layers as "embedding+lstm+2dnn", and I want to test the fp16 performance on TFS. 

By the way,I have tested that it can speed up 2times In pytorch on 2080Ti.

But when I do test on 2080Ti in TFS1.13GPU(cuda version=10.0),  fp16 is slower than float32 when batch is small, the test result as followings:

batch | 1080Ti(tfsv1.5) | 2080Ti(tfsv1.5) | 2080Ti(tfsv1.13)
    -  |float32 | fp16 | float32 | fp16 | float32 | fp16
50   | 31ms | 56ms | 48ms | 60ms | 51ms | 60ms
500 | 48ms | 42ms | 73ms | 62ms | 81ms | 61ms
1000 | 71ms | 42ms | 129ms | 60ms | 123ms | 58ms
1500 | 100ms | 47ms | 179ms | 76ms | 175ms | 60ms

I have checked that the CUDNN math type is CUDNN_TENSOR_OP_MATH, but I donot know if it has opened tensorcores or how to open it?

thanks a lot for any suggestions!

**Comment by wangyunxiaa** [comment]
@gadagashwini I have tested it on 2080Ti with tf1.13 and tf1.14, not using latest Tf nightly version.
my test code as followings:
`from tensorflow.python.platform import gfile
import time
import os
import datetime
os.environ['TF_ENABLE_AUTO_MIXED_PRECISION'] = '1'

with tf.Graph().as_default():
    output_graph_def = tf.GraphDef()
    with gfile.FastGFile("graph.pb", "rb") as f:
        output_graph_def.ParseFromString(f.read())
        _ = tf.import_graph_def(output_graph_def, name="")

    graph = tf.get_default_graph()
        with tf.Session() as sess:
        input_x = sess.graph.get_tensor_by_name("ac_input:0")
        state_in = sess.graph.get_tensor_by_name("Inference/pre_state:0")
        state_out = sess.graph.get_tensor_by_name("Inference/cur_state:0")
        out = sess.graph.get_tensor_by_name("Inference/final_output/output:0")`
the sess run and time collect as followings:
`feed = {
                 input_x: i2indataline50.reshape(2, -1),
                 state_in: pre_state.reshape(-1, all_hidden_num*50),
             }
        pre_state, output = sess.run([state_out, out], feed)
        print("=====loop 100 times=====")
        cur_time = time.time()
        for i in range(100):
            pre_state, output = sess.run([state_out, out], feed)
        now = (time.time() - cur_time)*1000
        print("t_use:", now/100)`
float32 and fp16 are two different graph.pb with the same model struct but different datetype(tf.float32 & tf.float16)
I am confused that pytorch is faster than tf, and fp16 is also speed up 2 times, what should i do to use tensorcores for fp16?
thanks!

**Comment by gadagashwini** [comment]
@wangyunxiaa, Will it possible to provide the graph.pb file to replicate the issue reported here. 
I tried with my own .pb file but I, end up in different error. Thanks!

**Comment by wangyunxiaa** [comment]
@gadagashwini , Hi,
The model and test code  in https://pan.baidu.com/s/1Q67Fvg8rlZappzjqIni_Ag

**Comment by ymodak** [comment]
Have you considered using [Automatic Mixed Precision for Deep Learning](https://developer.nvidia.com/automatic-mixed-precision) for your use case.

**Comment by ymodak** [comment]
Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by wangyunxiaa** [comment]
I have tried `export TF_ENABLE_AUTO_MIXED_PRECISION=1`
and `os.environ['TF_ENABLE_AUTO_MIXED_PRECISION'] = '1'` 
they are all no use

**Comment by wangyunxiaa** [comment]
@ymodak 
I am sorry for the latter reply,
Another question fused me ,
when I use tf.contrib.rnn.LSTMBlockFusedCell, It is about 2× times cost by tensorflow serving ( build from source 1.14 (GPU-2080TI cuda=10.1 version 1.13))than tensorflow(installed by pip install tensorflow-gpu==1.13.1)
I am confused that whether I have not add some build options or the blockfused op just speed up in python api not in c++ api?
the options I used is `bazel build --color=yes --curses=yes --config=cuda --copt="-fPIC" \
--copt=-O3 --copt=-msse4.1 --copt=-msse4.2 --copt=-mavx --copt=-mavx2 --copt=-mfma --copt=-mfpmath=both \
${TF_SERVING_BAZEL_OPTIONS} \
--verbose_failures \
--output_filter=DONT_MATCH_ANYTHING \
${TF_SERVING_BUILD_OPTIONS} \
tensorflow_serving/model_servers:tensorflow_model_server`
